### DISCUSSION TITLE
Project 2

### DISCUSSION THREAD

**Discussion by SqueezeU** [PEA-IA-DS-Proyecto-2/Project-2---Predict_Ratings_based_on_text_statements#2] [General]
Hey DreamTeam

I have uploaded the Project Task and the database.
Furthermore you find a pdf what I made which should help us to decide which tools we want to use.
I think it is easier to concentrate in a clear determination what we want to use and not always switch between the thousands possibilities which the profs mentioned during their classes.

Saludos
Errol

**Comment by edreivaj** [comment]
Hola Errol,

Gracias por tu trabajo. 

Puesto al día de todo lo que comentas  te lanzo la siguiente reflexión en relación a las herramientas a utilizar y quizá podemos asignar las tareas a realizar por cada uno.

Los  pasos a realizar en el proyecto y las notas y reflexiones son los siguientes:
1. ETL
    El software a utilizar es KNIME según el redactado del proyecto, aunque nos comentó Juan Diego que podíamos utilizar cualquiera de 
    similares características. Considerando esto y viendo tu cuadro comparativo mi opinión es utilizar KNIME, pues de tu cuadro se deriva que 
    también puede ser útil para otras tareas además de esta y por tanto podemos optar por reducir el numero de  herramientas que utilizamos

2. EDA
     Igual que el anterior.

3. Ética de la IA y Gobernanza de Datos
     Repasar las sesiones para hacer referencia a los puntos clave que debemos cumplir para ajustarnos a la norma GDPR y transcribirlo a 
     dentro de GitHub. (Decidir entre fichero de texto o pdf)

4. Desarrollo del modelo predictivo con herramientas AutoML
    En este punto hay que considerar algo que puede influir para elegir la herramienta. Hay que mirar si de todas las AutoML que  tenemos se 
    puede exportar a la herramienta de visualización de datos que elijamos en el  punto 6. Se me antoja pensar que herramientas del mismo 
    desarrollador no tendrán incompatibilidad en este sentido, pero conviene confirmarlo.

5. Visualización de datos con Power BI
    Igual que el punto  1 y 2 el enunciado es  claro, hay que utilizar  Power BI, aunque según Juan Diego podemos usar la que nos convenga.
    En este punto debemos decidir si vamos a contar activamente con Victor o no para elegir esta u otra herramienta. Según tú, Errol, Victor 
    trabaja con esta herramienta y facilitaría la elaboración del proyecto. En caso de que no pueda implicarse activamente podemos explorar 
    otras alternativas.

6. Entrega y documentación
    Como te comenté quizá podríamos aprovechar la potencia de WINDSURF y si tenemos tiempo hacer una web con enlaces a cada uno de 
    los puntos del proyecto.

Seguimos.
